Assets/Scripts/Sharing/DriveAccess.cs

```diff
@@ -99,6 +99,10 @@ private class ParentNotFound : Exception {
       case RuntimePlatform.OSXEditor:
         Debug.LogError("Host id not implemented for macOS");
         return "macOS-unknown";
+      case RuntimePlatform.LinuxPlayer:
+      case RuntimePlatform.LinuxEditor:
+        Debug.LogError("Host id is not implemented for Linux");
+        return "Linux-unknown";
```

Comment:
Note that having a working host id is required for proper working of the Google Drive backup system.

Reply:
Sure adding proper host id handling was on my todo list. The main goal here was to have a first version that compiles and that runs without exceptions